frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -263,9 +256,6 @@ export const OperandTableRow: React.FC<OperandTableRowProps> = ({
       <TableData className={tableColumnClasses[2]}>
         <OperandStatus operand={obj} />
       </TableData>
-      <TableData className={tableColumnClasses[3]}>
-        {_.get(obj.spec, 'version') || <div className="text-muted">Unknown</div>}
```

Comment:
@tlwu2013 Do you think we should change operand status to show `-` if there is no value? It might look similarly broken for CRDs where it's not expected.

Reply:
(Instead of `Unknown` which is what it shows today.)

Reply:
@spadgett That makes sense, thanks for catching this up!